cmd/cadvisor.go

```diff
@@ -111,6 +111,7 @@ var (
 		container.CPUTopologyMetrics:             struct{}{},
 		container.ResctrlMetrics:                 struct{}{},
 	}
+	ignoreSpecificMetrics DenyList = DenyList{}
```

Comment:
nit: comment what the deny list is / how it is different than `ignoreMetrics` / `ignoreWhitelist` above